CHANGELOG.md

```diff
@@ -31,6 +31,18 @@ Only use disabled form controls if research shows it makes the user interface ea
 
 This was added in [pull request #3187: Add disabled styles for form controls](https://github.com/alphagov/govuk-frontend/pull/3187).
 
+### Deprecated features
+
+#### Stop using the `govuk-button--disabled` class on buttons
+
+The `govuk-button--disabled` class is deprecated and will be removed in the next major release.
+
+If a button uses a `<button>` or `<input>` element, use the `disabled` HTML attribute instead. You will not need to make any changes if you're using the Button Nunjucks macro.
```

Comment:
Had a quick skim through the changelog and we tend to link to or "code up" the macro name

```suggestion
If a button uses a `<button>` or `<input>` element, use the `disabled` HTML attribute instead. You will not need to make any changes if you're using the [button](https://design-system.service.gov.uk/components/button/) Nunjucks macro.
```

```suggestion
If a button uses a `<button>` or `<input>` element, use the `disabled` HTML attribute instead. You will not need to make any changes if you're using the `govukButton` Nunjucks macro.
```

Reply:
I've modified this a little so that it's the Button documentation link in the first instance of the word, and the name of the macro in the second instance.